scout/image-details-view.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Image details view
+keywords: scout, supply chain, vulnerabilities, packages, cves, image, tag
+description: >
+  Docker Scout helps you understand your images and their dependencies
+---
+
+> **Note**
+>
+> Docker Scout is an [early access](../release-lifecycle.md#early-access-ea)
+> product, and requires a Pro, Team, or Business subscription.
+
+The image details view shows a breakdown of the Docker Scout analysis. You can
+access the image view both from within Docker Desktop, and from the image tag
+page on Docker Hub. This view provides a breakdown of the image hierarchy (base
+images), image layers, packages, and vulnerabilities.
+
+The image view lets you inspect the composition of an image from different
+perspectives. The view displays vulnerabilities and packages that an image
+contains. You can choose whether you want to see data for the image as a whole,
+or for a specific base image or layer.
+
+![The image details view in Docker Desktop](./images/dd-image-view.png){:width="700px"}
+
+## Image Hierarchy
+
+The image you inspect may have one or more base images listed under **Image
+hierarchy**. This means the author of the image used another image as a starting
```

Comment:
Again this may break formatting, GitHub doesn't show it, but may depend on MD renderer